libvips/resample/templates.h

```diff
@@ -402,6 +402,48 @@ double inline filter<VIPS_KERNEL_LANCZOS3>(double x)
 	return 0.0;
 }
 
+template <>
+double inline filter<VIPS_KERNEL_MKS2013>(double x)
+{
+	if (x < 0.0)
+		x = -x;
+
+	if (x >= 2.5)
+		return 0.0;
+
+	if (x >= 1.5)
+		return -0.125 * (x - 2.5) * (x - 2.5);
+
+	if (x >= 0.5)
+		return 0.25 * (4.0 * x * x - 11.0 * x + 7.0);
+
+	return 1.0625 - 1.75 * x * x;
```

Comment:
Nit: the non-optimized version is probably more readable:
```suggestion
	return 17.0 / 16.0 - 7.0 * x * x / 4.0;
```
(any C++ compiler will optimize it, so the assembly output should be equivalent)

Reference:
![mks-2013-formula](https://johncostella.com/magic/mks-2013-formula.png)